webpack/components/AnsibleHostDetail/components/JobsTab/JobsTabHelper.js

```diff
@@ -24,3 +27,40 @@ export const fetchPreviousFn = fetchJobsFn(previousJobsSearch);
 export const renameData = data => ({
   jobs: data.jobInvocations.nodes,
 });
+
+export const joinErrors = errors => errors.map(err => err.message).join(', ');
+
+const formatError = error =>
+  sprintf(
+    __('There was a following error when deleting Ansible config job: %s'),
+    error
+  );
+
+const onError = error => {
+  showToast({ type: 'danger', message: formatError(error) });
+};
+
+const onCompleted = data => {
+  const { errors } = data.cancelRecurringLogic;
```

Comment:
can`cancelRecurringLogic` be undefeind?

Reply:
I do not think so, that is what is always sent when mutation completes - unless there is network error which is handled by `onError`